stake-pool/py/stake_pool/actions.py

```diff
@@ -589,3 +589,60 @@ async def decrease_validator_stake(
     signers = [payer, staker] if payer != staker else [payer]
     await client.send_transaction(
         txn, *signers, opts=TxOpts(skip_confirmation=False, preflight_commitment=Confirmed))
+
+
+async def create_token_metadata(client: AsyncClient, payer: Keypair, stake_pool_address: PublicKey,
+                                name: str, symbol: str, uri: str):
+    resp = await client.get_account_info(stake_pool_address, commitment=Confirmed)
+    data = resp['result']['value']['data']
+    stake_pool = StakePool.decode(data[0], data[1])
+
+    (withdraw_authority, _seed) = find_withdraw_authority_program_address(STAKE_POOL_PROGRAM_ID, stake_pool_address)
+
+    txn = Transaction()
+    txn.add(
+        sp.create_token_metadata(
+            sp.CreateTokenMetadataParams(
+                program_id=STAKE_POOL_PROGRAM_ID,
+                stake_pool=stake_pool_address,
+                manager=stake_pool.manager,
+                pool_mint=stake_pool.pool_mint,
+                payer=payer.public_key,
+                name=name,
+                symbol=symbol,
+                uri=uri,
+                withdraw_authority=withdraw_authority,
+                token_metadata=TOKEN_PROGRAM_ID,
```

Comment:
`token_metadata` should be the result of `find_metadata_account(stake_pool.pool_mint)`, and then you still need to add the system program id and token metadata program id